docs/result.md

```diff
@@ -1,14 +1,209 @@
+    },
+    {
+      'job_type': 'download',
+      'url':  'https://api.data.gov/regulations/v3/download?documentId=EPA-HQ-OAR-2011-0028-0108&contentType=html'
+    }
+  ]
+}
+```
+
+* Documents
+```
+{
+  'client_id': '123ABC',
+  'job_id': 'ABC123`,
+  'data': {
+    "documents": [
+      {
+        'agency_folder': {
+          'name': 'CMS'
+          'docket_folder': {
+            'name': 'CMS-2005-0001'
+            'document_folder': {
+              'name': 'CMS-2005-0001-0001'
+              'file': {
+                'name': 'basic_document.json'
+                'json': {
+                  "agencyAcronym": "CMS",
+                  "allowLateComment": false,
+                  "attachmentCount": 0,
+                  "commentDueDate": "2005-10-03T23:59:59-04:00",
+                  "commentStartDate": "2005-08-04T00:00:00-04:00",
+                  "docketId": "CMS-2005-0001",
+                  "docketTitle": "Medicare Program; Revised Civil Money Penalties, Assessments, Exclusions, and Related Appeals Procedures",
+                  "docketType": "Nonrulemaking",
+                  "documentId": "CMS-2005-0001-0001",
+                  "documentStatus": "Posted",
+                  "documentType": "Proposed Rule",
+                  "frNumber": "05-15291",
+                  "numberOfCommentsReceived": 0,
+                  "openForComment": false,
+                  "postedDate": "2005-08-04T00:00:00-04:00",
+                  "title": "Medicare Program; Revised Civil Money Penalties, Assessments, Exclusions, and Related Appeals Procedures"
+                }
+              }
+            }
+          }
+        }
+      },
+      
+                ...
+
+      {
+        'agency_folder': {
+          'name': 'FMCSA'
+          'docket_folder': {
+            'name': 'FMCSA-1997-2350'
+            'document_folder': {
+              'name': 'FMCSA-1997-2350-21655'
+              'file': {
+                'name': 'basic_document.json'
+                'json': {
+                  "agencyAcronym": "FMCSA",
+                  "allowLateComment": false,
+                  "attachmentCount": 1,
+                  "commentDueDate": "2000-12-15T23:59:59-05:00",
+                  "commentStartDate": null,
+                  "docketId": "FMCSA-1997-2350",
+                  "docketTitle": "Notice of Proposed Rulemaking (NPRM) - Hours of Service of Drivers",
+                  "docketType": "Rulemaking",
+                  "documentId": "FMCSA-1997-2350-21655",
+                  "documentStatus": "Posted",
+                  "documentType": "Public Submission",
+                  "numberOfCommentsReceived": 1,
+                  "openForComment": false,
+                  "postedDate": "2000-10-17T00:00:00-04:00",
+                  "rin": "2126-AA23",
+                  "title": "Lee & Jo Batton - Comments"
+                }
+              }
+            }
+          }
+        }
+      }
+    ]
+  }
```

Comment:
I think that all versions should have the `data` field be an array.  If there is only one piece of data, then it is fine to have an array with a single element.